packages/tools/src/tools/segmentation/BrushTool.ts

```diff
@@ -398,75 +434,102 @@ class BrushTool extends BaseTool {
 
   public invalidateBrushCursor() {
     if (this._hoverData !== undefined) {
-      const { data } = this._hoverData.brushCursor;
-
-      data.invalidated = true;
+      const annotation = getAnnotation('brushCursorUID');
+      annotation.data.invalidated = true;
     }
   }
 
+  public isPointNearTool(): boolean {
+    return false;
+  }
+
   renderAnnotation(
     enabledElement: Types.IEnabledElement,
     svgDrawingHelper: SVGDrawingHelper
-  ): void {
+  ): boolean {
+    const renderStatus = false;
     if (!this._hoverData) {
       return;
     }
 
     const { viewport } = enabledElement;
+    const { element } = viewport;
 
     const viewportIdsToRender = this._hoverData.viewportIdsToRender;
 
     if (!viewportIdsToRender.includes(viewport.id)) {
       return;
     }
 
-    const brushCursor = this._hoverData.brushCursor;
+    let annotations = getAnnotations(this.getToolName(), viewport.element);
 
-    if (brushCursor.data.invalidated === true) {
-      const { centerCanvas } = this._hoverData;
-      const { element } = viewport;
+    if (!annotations?.length) {
+      return renderStatus;
+    }
+
+    annotations = this.filterInteractableAnnotationsForElement(
+      element,
+      annotations
+    );
 
+    if (!annotations?.length) {
+      return renderStatus;
+    }
+
+    const annotation = annotations[0];
+
+    if (annotation.data.invalidated === true) {
```

Comment:
It is not clear to me what the purpose of this entire `if` is. Please add a comment.